portfolio/src/main/java/com/google/sps/servlets/ContactMeServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/contact-me")
+public class ContactMeServlet extends HttpServlet{
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String email = request.getParameter("email");
+        String message = request.getParameter("message");
+
+        String log = "Email: " + email + "\nMessage: " + message;
+
+        System.out.println(log);
```

Comment:
This is beyond the scope of SPS, but at Google we use a logging library called Flogger.

The syntax would be something like logger.atInfo().log("Email: %s\nMessage: %s", email, message);

https://github.com/google/flogger

Reply:
Here's the link for viewing your logs: https://sites.google.com/sps-program.com/students/technical-resources/im-stuck-debugging-common-obstacles#h.p_ommhVhAe-bNK